test/elastic_record/as_document_test.rb

```diff
@@ -30,31 +30,31 @@ def test_as_partial_update_document
 
   class SpecialFieldsModel
     include TestModel
-    attr_accessor :author, :book_length, :commenters, :meta,
+    define_attributes :author, :book_length, :commenters, :meta
 
     class Author
       include TestModel
-      attr_accessor :name, :salary_estimate
+      define_attributes :name, :salary_estimate
```

Comment:
SpecialFieldsModel wasn't correctly implemented, so dirty tracking wasn't working.

Reply:
A quick review tells me we could cleanup the models used for testing.